src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/shared-module/components/aggregation/aggregation.component.html

```diff
@@ -56,14 +56,22 @@
         [selected]="item.active"
         [value]="item"
       >
-        <div
-          class="aggregation__option-title"
-          matTooltip="{{ item.getDisplayName(locale) }}"
-          [matTooltipDisabled]="false"
-        >
-          <div class="aggregation__option-name">{{ item.getDisplayName(locale) }}</div>
-          <div class="aggregation__option-number">{{ item.count }}</div>
-        </div>
+        <ng-container *ngIf="item.getDisplayName(locale) as displayName; else noValue">
```

Comment:
Evaluate a way to not use the displayName in the view (maybe use a displayname getter in data contract)